pkg/pod/pod.go

```diff
@@ -223,6 +223,31 @@ func (builder *Builder) DeleteAndWait(timeout time.Duration) (*Builder, error) {
 	return builder, nil
 }
 
+// DeleteImmediate removes the pod immediately object and resets the builder object.
+func (builder *Builder) DeleteImmediate() (*Builder, error) {
+	if valid, err := builder.validate(); !valid {
+		return builder, err
+	}
+
+	glog.V(100).Infof("Immediately deleting pod %s in namespace %s",
+		builder.Definition.Name, builder.Definition.Namespace)
+
+	if !builder.Exists() {
+		return builder, fmt.Errorf("pod cannot be deleted because it does not exist")
```

Comment:
We just return nil if it doesn't exist instead of an error

Reply:
The way the pod Delete method is currently written is outdated (you don't have to change it)

Reply:
tnx, updated.